libs/barista-components/alert/BUILD.bazel

```diff
@@ -1,7 +1,30 @@
-load("//tools/bazel_rules:index.bzl", "stylelint")
+load("@io_bazel_rules_sass//:defs.bzl", "sass_binary", "sass_library")
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("//tools/bazel_rules:index.bzl", "jest", "ng_module_view_engine", "stylelint")
 
 package(default_visibility = ["//visibility:public"])
 
+ng_module_view_engine(
+    name = "compile",
+    srcs = glob(
+        include = ["**/*.ts"],
+        exclude = [
+            "**/*.spec.ts",
+            "src/test-setup.ts",
+        ],
+    ),
+    angular_assets = [
+        ":styles",
+        ":src/alert.html",
+    ],
+    module_name = "@dynatrace/barista-components/alert",
+    tsconfig = "tsconfig_lib",
+    deps = [
+        "//libs/barista-components/icon:compile",
+        "@npm//@angular/core",
```

Comment:
Do we have to list the common module here as well? 
It's imported inside the alert.module.ts

Reply:
good catch
Once we do not have to use the view engine rule we get strict dependency checks where a missing third party dep would break the build. 
To be complete: The most important deps are the ones from within the codebase, because those actually determine what needs to be built. If we get those "relative" deps wrong, the build will fail (also with this version already)

Reply:
@furtnerl I created the `ng_module_view_engine` macro exactly with the same API as the `ng_module` (the ts_library with the angular plugin). So to check for the dependencies just switch the `ng_module_view_engine` with the `ng_module` and you have checks. 

After the deps checks passed then revert to the view engine one.